roles/ceph-grafana/tasks/configure_grafana.yml

```diff
@@ -34,9 +34,25 @@
     - "/etc/grafana/provisioning/dashboards"
     - "/etc/grafana/provisioning/notifiers"
 
+- name: get dashboard path for github
+  set_fact:
+    _grafana_dashboard_github_path: "{{ grafana_dashboard_version }}/monitoring/ceph-mixin/dashboards_out"
+  when:
+    - grafana_dashboard_version == 'pacific' or grafana_dashboard_version == 'quincy'
+    - not containerized_deployment | bool
+    - not ansible_facts['os_family'] in ['RedHat', 'Suse']
+
+- name: get dashboard path for github
+  set_fact:
+    _grafana_dashboard_github_path: "{{ grafana_dashboard_version }}/monitoring/grafana/dashboards"
+  when:
+    - grafana_dashboard_version != 'pacific' or grafana_dashboard_version != 'quincy'
+    - not containerized_deployment | bool
+    - not ansible_facts['os_family'] in ['RedHat', 'Suse']
+
 - name: download ceph grafana dashboards
   get_url:
-    url: "https://raw.githubusercontent.com/ceph/ceph/{{ grafana_dashboard_version }}/monitoring/grafana/dashboards/{{ item }}"
+    url: "https://raw.githubusercontent.com/ceph/ceph/{{ _grafana_dashboard_github_path }}/{{ item }}"
```

Comment:
```suggestion
    url: "https://raw.githubusercontent.com/ceph/ceph/{{ grafana_dashboard_version }}/monitoring/ceph-mixin/dashboards_out/{{ item }}"
```

Reply:
The change I proposed would be backwards compatible as the dashboards for octopus and earlier life in the old location still.

Reply:
I mean, we could also change this like that within master and cherry-pick into stable-6.0 onwards. Works for me. Is that the direction to go?

Reply:
> I mean, we could also change this like that within master and cherry-pick into stable-6.0 onwards. Works for me. Is that the direction to go?

yes

Reply:
> The change I proposed would be backwards compatible as the dashboards for octopus and earlier life in the old location still.

the branch 'main' isn't intended to be used for deploying stables releases.

Reply:
Changed.